cluster/multus-daemonset.yml

```diff
@@ -188,8 +144,40 @@ spec:
               mountPath: /host/etc/cni/net.d
             - name: cnibin
               mountPath: /host/opt/cni/bin
-            - name: multus-cfg
-              mountPath: /tmp/multus-conf
+      initContainers:
+        - name: install-multus-binary
+          image: ghcr.io/k8snetworkplumbingwg/multus-cni:thick
```

Comment:
Do you think it's worth mentioning in commit message that this change switches to a thick image?

Reply:
Sure, that may become useful in the future. Lemme fix it

Reply:
Fixed